React/Views/ScrollView/RCTScrollView.m

```diff
@@ -549,6 +549,13 @@ - (void)setAccessibilityLabel:(NSString *)accessibilityLabel
 {
   [_scrollView setAccessibilityLabel:accessibilityLabel];
 }
+
+- (void)setAccessibilityRole:(NSAccessibilityRole)accessibilityRole
+{
+    if (@available(macOS 10.10, *)) {
```

Comment:
Do we really need a 10.10 availability check?

Reply:
It wouldn't compile without it :/ https://developer.apple.com/documentation/appkit/nsaccessibility/1535005-accessibilityrole

Reply:
This feels.. wrong. We should be able to compile without this...

Reply:
I lied, it does compile without the availability check! Thanks for catching this :)